include/openssl/x509_vfy.h.in

```diff
@@ -72,11 +72,76 @@ typedef enum {
     .generate_stack_macros("X509_VERIFY_PARAM");
 -}
 
+/* This is used for a table of trust checking functions */
```

Comment:
With the amount moved here, I think it would be prudent to have
```C
#include <openssl/x509_vfy.h>
```
in ` include/openssl/x509.h.in `

Reply:
It is there already - not at the top of the file as one would expect though.